snapcraft.yaml

```diff
@@ -1406,7 +1406,14 @@ parts:
         # Build the static website
         make doc
 
-        # Copy the static website
+        # Remove unneeded bits
+        rm doc/html/objects.inv    # only objects.inv.txt is used
+        # not needed once built
+        rm doc/html/.buildinfo
+        rm -rf doc/html/_sphinx_design_static/
+        rm -rf doc/html/_sources/
```

Comment:
The sources are linked at the bottom of each page, so this will cause broken links in the footer.

Reply:
Alternatively, we could probably find some way to use a different footer for snap builds. Not sure if the sources are needed for everyone (I would like to keep them for the online docs and normal local build though, since they are very useful when working with the docs).

Reply:
Can you link to the online ones always?

Reply:
This would be error-prone since the doc version shipped with the snap might differ from what is online - so the sources might not correspond to the documentation shipped with the snap.

Reply:
got it, makes sense, thanks

Reply:
Thanks for catching this @ru-fu. #313 stops removing those sources.